pyluba/bluetooth/ble_message.py

```diff
@@ -1052,4 +1054,210 @@ def _getPackageType(self, typeValue: int):
         return typeValue & 3
 
     def _getSubType(self, typeValue: int):
-        return (typeValue & 252) >> 2
\ No newline at end of file
+        return (typeValue & 252) >> 2
+
+# ==================================================================
+
+    # def send_order_msg_net(self, dev_net, logtype, is_iotable):
+    #     proto_buf_builder_set = self.get_proto_buf_builder_set(dev_net_pb2.MsgCmdType.MSG_CMD_TYPE_ESP, dev_net_pb2.MsgDevice.DEV_COMM_ESP, dev_net_pb2.MsgAttr.MSG_ATTR_REQ)
+    #     proto_buf_builder_set.net.CopyFrom(dev_net)
+    #     self.send_msg(proto_buf_builder_set, logtype, is_iotable)
+
+    async def send_order_msg_net(self, build):
+        luba_msg = luba_msg_pb2.LubaMsg(
+            msgtype=luba_msg_pb2.MsgCmdType.MSG_CMD_TYPE_ESP,
+            sender=luba_msg_pb2.MsgDevice.DEV_MOBILEAPP,
+            rcver=luba_msg_pb2.MsgDevice.DEV_COMM_ESP,
+            msgattr=luba_msg_pb2.MsgAttr.MSG_ATTR_REQ,
+            seqs=1,
+            version=1,
+            subtype=1,
+            net=build)
+
+        byte_arr = luba_msg.SerializeToString()
+        await self.postCustomDataBytes(byte_arr)
+
+    def get_4g_module_info(self):
+        build = dev_net_pb2.DevNet(
+            todev_get_mnet_cfg_req=dev_net_pb2.DevNet().todev_get_mnet_cfg_req)
+        print("Send command -- Get device 4G network module information")
+        self.send_order_msg_net(build, 96, True)
+
+    # def get_4g_module_info(self):
+    #     build = dev_net_pb2.DevNet(
+    #         todev_get_mnet_cfg_req=dev_net_pb2.DevNet.TodevGetMnetCfgReq())
+    #     self.send_order_msg_net(build)
+
+    # ---------------------------
+
+    def get_4g_info(self):
+        build = dev_net_pb2.DevNet(
+            TodevMnetInfoReq=dev_net_pb2.DevNet().TodevMnetInfoReq)
+        print("Send command -- Get device 4G network information")
+        self.send_order_msg_net(build, 95, True)
+
+    def set_zmq_enable(self):
+        build = dev_net_pb2.DevNet(
+            todev_set_dds2_zmq=dev_net_pb2.DrvDebugDdsZmq(
+                is_enable=True,
+                rx_topic_name="perception_post_result",
+                tx_zmq_url="tcp://0.0.0.0:5555"
+            )
+        )
+        print("Send command -- Set vision ZMQ to enable")
+        self.send_order_msg_net(build, 97, True)
+
+    def set_iot_setting(self, iot_conctrl_type: dev_net_pb2.iot_conctrl_type):
+        build = dev_net_pb2.DevNet(TodevSetIotOfflineReq=iot_conctrl_type)
+        print("Send command -- Device re-online")
+        self.send_order_msg_net(build, 961, True)
+
+    def send_to_dev_ble_sync(self):
+        build = dev_net_pb2.DevNet(todev_ble_sync=2)
+        print("Send Bluetooth heartbeat==sendTodevBleSync==type:2")
+        self.send_order_msg_net(build, 39, False)
+
+    def send_to_dev_ble_sync(self, current_client, type):
```

Comment:
so this command is already written, however this is a bit tidier than the original function.

probably need to look at using a debug logger over print and adding method definition comments.